### DISCUSSION TITLE
How to use this with Fabric mods?

### DISCUSSION THREAD

**Discussion by Dorzalty** [IrisShaders/Iris#606] [Q&A] [answered]
Hello!

I'm not sure if I'm understanding this all at all, but hopefully someone can help me out. Is this something that can work alongside other mods, specifically Fabric mods? If so, how do I install this and use other mods (minimap mods etc) with it? I'd really love to run some shaders + get a performance boost (which I heard this grants via Sodium), but Optifine--sadly--runs really bad for me and I don't think I have a bad PC at all. lol 

Any help could be greatly appreciated! Thank you for reading!

**Minecraft Version:** 1.17.1
**Mods In Use:** Xaero's World Map + Mini Map and Fabric + Fabric API

**Comment by forkiesassds** [comment]
VoxelMap works fine for me. Did you install the Fabric mod loader tho?

**Comment by byquanton** [comment, marked as answer]
Just download Iris from Modrinth or the GitHub Releases and Put it into your mods Folder. The Modrinth Version of Iris contains Sodium
https://modrinth.com/mod/Iris
